flask_project/campaign_manager/templates/create_campaign.html

```diff
@@ -64,14 +64,14 @@
                                     <div class="form-header col-md-6">
                                         <div class="form-group required-form">
                                             <label class="panel-header-info" for="start_date">START DATE</label>
-                                            {{ form.start_date }}
+                                            {{ form.start_date(readonly='true') }}
```

Comment:
maybe you could update form
https://github.com/ann26/field-campaigner/blob/d3867c4ad60d08fe79b8d4ce9c5ff9c869f588a8/flask_project/campaign_manager/forms/campaign.py#L65